.prettierignore

```diff
@@ -0,0 +1,27 @@
+/build
```

Comment:
This should be based on the project's gitignore. It looks like it is copied from another project.

Reply:
Yes I copied it from element-web 😅. I now replaced it with the contents of the `.gitignore`.